src/lib/logger.test.ts

```diff
@@ -418,6 +418,130 @@ describe("authed logger", () => {
       Logger.logCommentEvent(deleteEventPayload);
     });
 
+    it("can log a EXPAND tile comment thread event", (done) => {
+      const document = createDocumentModel({
+        type: ProblemDocument,
+        uid: "1",
+        key: "source-document",
+        createdAt: 1,
+        content: {},
+        visibility: "public"
+      });
+      stores.documents.add(document);
+      const tile = ToolTileModel.create({ content: TextContentModel.create() });
+      const tileId = tile.id;
+      const documentKey = document.key;
+      const expandCommentPayload: ILogComment = {
+        focusDocumentId: document.key,
+        focusTileId: tile.id,
+        isFirst: false,
+        commentText: '',
+        action: "expand"
+      };
+
+      mockXhr.post(/.*/, (req, res) => {
+        const expand = JSON.parse(req.body());
+        expect(expand.event).toBe("EXPAND_COMMENT_THREAD_FOR_TILE");
+        expect(expand.parameters.tileId).toBe(tileId);
+        expect(expand.parameters.documentKey).toBe(documentKey);
+        done();
+        return res.status(201);
+      });
+      Logger.logCommentEvent(expandCommentPayload);
+    });
+    
+    it("can log a COLLAPSE tile comment thread event", (done) => {
+      const document = createDocumentModel({
+        type: ProblemDocument,
+        uid: "1",
+        key: "source-document",
+        createdAt: 1,
+        content: {},
+        visibility: "public"
+      });
+      stores.documents.add(document);
+      const tile = ToolTileModel.create({ content: TextContentModel.create() });
+      const tileId = tile.id;
+      const documentKey = document.key;
+      const collapseCommentPayload: ILogComment = {
+        focusDocumentId: document.key,
+        focusTileId: tile.id,
+        isFirst: false,
+        commentText: '',
+        action: "collapse"
+      };
```

Comment:
Lines 422-440 are almost the same as 454-472. Should they be combined into a function? Seems like at least creating a default document could be turned into a useful function.

Reply:
Good point! That made me notice that the comment tests were mixed in with tile tests so I separated out the comment ones and added the util for making a document since those were exactly the same. I left the stuff in the mock callback out since I would have had to pass 3 of the 4 things it checks in as parameters. I don't have strong opinions though so happy to if you'd like it better!